src/clandinin_lab_to_nwb/brezovec/brezovecbehaviorinterface.py

```diff
@@ -1,22 +1,245 @@
+        "lab_absolute_rotation_vector_z": "Absolute orientation of the sphere represented as rotation angle/axis (radians) in laboratory coordinates (z).",
+        "lab_integrated_x_y_position_x": "Integrated x position (radians) in laboratory coordinates. Scale by sphere radius for true position.",
+        "lab_integrated_x_y_position_y": "Integrated y position (radians) in laboratory coordinates. Scale by sphere radius for true position.",
+        "lab_integrated_animal_heading": "Integrated heading orientation (radians) of the animal in laboratory coordinates. This is the direction the animal is facing.",
+        "lab_animal_movement_direction": "Instantaneous running direction (radians) of the animal in laboratory coordinates. This is the direction the animal is moving in the lab frame (add to animal heading to get direction in the world).",
+        "animal_movement_speed": "Instantaneous running speed (radians/frame) of the animal. Scale by sphere radius for true speed.",
+        "integrated_forward_side_motion_x": "Integrated x position (radians) of the sphere in laboratory coordinates neglecting heading. Equivalent to the output from two optic mice.",
+        "integrated_forward_side_motion_y": "Integrated y position (radians) of the sphere in laboratory coordinates neglecting heading. Equivalent to the output from two optic mice.",
+        "timestamp": "Either position in video file (ms) or frame capture time (ms since epoch).",
+        "sequence_counter": "Position in the current frame sequence. Usually corresponds directly to the frame counter, but can reset to 1 if tracking is reset.",
+        "delta_timestamps": "Time (ms) since the last frame.",
+        "alt_timestamp": "Frame capture time (ms since midnight).",
+    }
+
+    int_type_columns = ["sequence_counter", "frame_counter"]
+
+    def __init__(self, file_path: str):
+        self.file_path = Path(file_path)
+        assert self.file_path.is_file(), f"FIle path does not exist: {self.file_path}"
 
-    def __init__(self):
         # This should load the data lazily and prepare variables you need
         pass
 
-    def get_metadata(self):
+    def get_metadata(self) -> DeepDict:
         # Automatically retrieve as much metadata as possible
         metadata = super().get_metadata()
 
         return metadata
 
-    def add_to_nwb(self, nwbfile: NWBFile, metadata: dict):
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict):
         # All the custom code to write to PyNWB
 
+        descriptions = BrezovecBehaviorInterface.column_to_description
+        columns = list(descriptions)[:23]
+
+        int_type_columns = BrezovecBehaviorInterface.int_type_columns
+        column_to_type = lambda x: pl.Int64 if x in int_type_columns else pl.Float64
+
+        polars_schema = {column: column_to_type(column) for column in columns}
+        df_fitrac = pl.scan_csv(
```

Comment:
How big are these CSV files? Usually they can be read in memory with no problem, but I support it's possible to have a file big enough where lazy reading would be necessary. Would it be possible to add an implementation option that does not require polars?